src/librustdoc/passes/collect_intra_doc_links.rs

```diff
@@ -1575,17 +1575,17 @@ fn resolution_failure(
                         _ => None,
                     };
                     // See if this was a module: `[path]` or `[std::io::nope]`
-                    if let Some(module) = last_found_module {
-                        let module_name = collector.cx.tcx.item_name(module);
+                    if let Some(_module) = last_found_module {
                         let note = format!(
-                            "the module `{}` contains no item named `{}`",
-                            module_name, unresolved
+                            "there is no item named `{}` in scope",
+                            unresolved
                         );
                         if let Some(span) = sp {
                             diag.span_label(span, &note);
                         } else {
                             diag.note(&note);
                         }
+                        diag.help(&format!("did you mean to import `{}`?", unresolved));
```

Comment:
I don't think this is very helpful is we don't suggest where to import it _from_. It's just as likely this was a typo or a renamed method.

Reply:
See my comment [above](https://github.com/rust-lang/rust/pull/77469#discussion_r499061527) :)

I preemptively removed it locally; I'll amend the commit.

Reply:
Actually I'll make a new commit, and squash at the end so it's easy to review.